### DISCUSSION TITLE
Can I pgcenter record at a particular time?

### DISCUSSION THREAD

**Discussion by sillycube** [lesovsky/pgcenter#105] [General]
I would like to know the active connections at a particular time, say 11:30am - 12:00nn UTC+0:00. It's the peak hour of my app.

Since my timezone is UTC+8:00, I can't check it manually with `pgcenter top`. Is there any way I can run `pgcenter record` and check the active connections later?

Also, how can I check the active connection? I can't find the `activity: 2/100 conns` from pg_stat_activity. Though I can see it with `pgcenter top`

**Comment by lesovsky** [comment]
You can use `cron` utility and run "pgcenter record" with any schedule you want. But pgcenter doesn't record summary information which provided by `top`, you just can output information recorded from pg_stat_activity (with pgcenter report -A) and count using other shell utilities like `sort`, `uniq`, etc.

**Reply by sillycube** [reply]
Thanks, I'll take a look!
